django/src/rdwatch/server/settings.py

```diff
@@ -21,12 +21,19 @@ class BaseConfiguration(Configuration):
     WSGI_APPLICATION = 'rdwatch.server.application'
     ALLOWED_HOSTS = ['*']
     DEBUG = values.BooleanValue(False, _environ_prefix='RDWATCH_DJANGO')
```

Comment:
Not sure what is going on, but this isn't evaluating to True when RDWATCH_DJANGO_DEBUG is set to 1 on my system.  Looking at all os.environ I can see it in there, but this is always returning false;
```suggestion
    DEBUG = 'RDWATCH_DJANGO_DEBUG' in os.environ
```

Reply:
My bad, I fixed this locally but didn't push it here - I opened a PR on top of this branch with the fix #236 as its unrelated to this PR.

Reply:
Marking as resolved, your other PR clears it up.